chefignore

```diff
@@ -104,3 +104,9 @@ Strainerfile
 ###########
 .vagrant
 Vagrantfile
+
+# GitHub #
```

Comment:
This should get removed. We use these files and they should be managed in git

Reply:
Removed in amended commit